hybrid/SampleApps/NoteSync/NoteSync/SFContentSoqlSyncDownTarget.m

```diff
@@ -274,24 +290,12 @@ + (SFContentSoqlSyncDownTarget*) newSyncTarget:(NSString*)query {
 }
 
 
-#pragma mark - From/to dictionary
-
-+ (SFContentSoqlSyncDownTarget*) newFromDict:(NSDictionary*)dict {
```

Comment:
There's only one target factory method now for sync down and sync up.  Each calls the `initWithDict` initializer of any of respective child classes they manage.